### PR TITLE
return different return value than 0 when build fails

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-8521

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [ ] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [ ] Pull request includes link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: osbs/cli/main.py [hunk 1]
```diff
@@ -317,13 +317,18 @@ def _print_build_logs(args, osbs, build):
             logger.error("Error during fetching logs for build %s: %s", build_id, repr(ex))
 
         osbs.wait_for_build_to_finish(build_id)
-        _display_build_summary(osbs.get_build(build_id))
+        return _display_build_summary(osbs.get_build(build_id))
```
Comment (MartinBasti) [inline osbs/cli/main.py]: Return should stay here, or new `if` block is wrongly intended

Comment (MartinBasti) [reply osbs/cli/main.py]: indented

Comment (MartinBasti) [reply osbs/cli/main.py]: nevermind, I'm cannot read diffs

**Comment by MartinBasti** [comment]
Typo in commit msg `reuturn`

**Comment by rcerven** [comment]
typo fixed

**Comment by athos-ribeiro** [comment]
We have not been testing the cli module. Hence, waiving code coverage requirement.

Since this should be an expected behavior from the cli, we should also waive the docs PR requirement, although it would be nice to have an entry in the release notes for this one.

**Comment by rcerven** [comment]
rn updated: - return different return value than 0 when build fails (-1)

**Pull request merged**